langchain/src/prompts/tests/selectors.test.ts

```diff
@@ -0,0 +1,40 @@
+import { expect, test } from "@jest/globals";
+import { FakeEmbeddings } from "../../embeddings/fake.js";
+import { LengthBasedExampleSelector } from "../selectors/LengthBasedExampleSelector.js";
+import { SemanticSimilarityExampleSelector } from "../selectors/SemanticSimilarityExampleSelector.js";
+import { HNSWLib } from "../../vectorstores/hnswlib.js";
```

Comment:
@nfcampos now using HNSWLib! test passes

Reply:
Great, thank you!